### PR TITLE
{bio}[GCC/8.2.0-2.31.1] MUSCLE v3.8.1551

### PR DESCRIPTION
**Pull request by omula**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/m/MUSCLE/MUSCLE-3.8.1551_fix-mk-hardcoding.patch [hunk 1]
```diff
@@ -0,0 +1,12 @@
+diff -Naur /dev/shm/MUSCLE/3.8.1551/GCC-8.2.0-2.31.1.orig/Makefile /dev/shm/MUSCLE/3.8.1551/GCC-8.2.0-2.31.1/Makefile
```
Comment (boegel) [inline easybuild/easyconfigs/m/MUSCLE/MUSCLE-3.8.1551_fix-mk-hardcoding.patch]: @omula Please add a short description + author or upstream reference on top (& adjust the checksum accordingly)

Comment (omula) [reply easybuild/easyconfigs/m/MUSCLE/MUSCLE-3.8.1551_fix-mk-hardcoding.patch]: Done :)

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node3102.skitty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/555899ef48063ec9e8fcb87f639f6259 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2710.swalot.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/8de37984b6dba7debfde8abcb9c3dd00 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
generoso - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2660 v4 @ 2.00GHz, Python 3.6.8
See https://gist.github.com/ed1f45d204e4c1b307a67116c6c6a057 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @omula!

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node3107.skitty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/af7dc2c80a6443c7b234659023ef6adb for a full test report.

**Pull request merged**